### ISSUE TITLE
dynamic: "create thread bypassing process freeze" FPs

### ISSUE THREAD

**Issue by mike-hunhoff** [mandiant/capa-rules#1023]
I'm seeing many FPs for _create thread bypassing process freeze_ for capa dynamic anlaysis of CAPE sandbox, e.g.:
```
  NtCreateThreadEx(
    ThreadHandle=0x3f8,
    ProcessHandle=0xffffffff,
    StartAddress=0x100100,
    Parameter=0x40,
    CreateFlags=0x1,
    ThreadId=0x1df8,
    ProcessId=0x1a70,
    Module="example.exe",
  ) -> 0x0
```

capa is detecting `0x40` constant passed via _Parameter_ but we only want to match if it is passed via _CreateFlags_.

**Comment by dhruvak001** [comment]
Even though with that I have properly nested the code, it still reads them like:
```
  features:
    - and:
      - api: ntdll.NtCreateThreadEx
      properties:
        arguments:
          CreateFlags: 0x40
``` 

Instead of: 
```
  features:
    - and:
      - api: ntdll.NtCreateThreadEx
        properties:
          arguments:
            CreateFlags: 0x40
```
